chat/be/server.js

```diff
@@ -64,7 +64,7 @@ async function run() {
     const user = await User.findOne({ email, password });
 
     if (user) {
-      res.cookie('user', user._id);
+      res.cookie('user', user._id, { maxAge: 31556926000 });
```

Comment:
Can this be reformatted as 365 * 24 * 60 * 60 * 1000 to better clarify how long it is, and what units?